lib/client/index.js

```diff
@@ -141,12 +141,13 @@ var generateGUID = function (){
 var uuidSource = fs.readFileSync(require.resolve('node-uuid'), 'utf-8');
 
 responder.source = [
-	';(function (Primus, undefined) {',
```

Comment:
This has been removed because Primus already wraps the plugin library in an IIFE.